lib/generic/Pole.hpp

```diff
@@ -5,7 +5,7 @@
 #include "ColordanceTypes.hpp"
 #include "Grid.hpp"
 
-static constexpr uint8_t kLightsPerPole = 45;
+constexpr uint8_t kLightsPerPole = 45;
```

Comment:
I would mark this `inline` in addition to removing `static`.

Reply:
I'm not sure about the spec (I can't find docs to refute it) but it doesn't seem like `inline` and  `constexpr` play nicely in this context.

Reply:
Yeah, that seems to be an issue with our compiler. However, you should still include the `static` if possible - without it, the compiler might include one copy per instance of this class (not sure if that's optimized).